Dockerfile

```diff
@@ -0,0 +1,58 @@
+FROM golang:1.14.4-buster
+LABEL maintainer="Alok Nerurkar <alok@stream.space>"
```

Comment:
You can make yourself the maintainer.

Which includes perennial responsibility of this file!